model/src/cfd.rs

```diff
@@ -1425,6 +1425,8 @@ impl Cfd {
 
         let hours_left = settlement_time - OffsetDateTime::now_utc();
 
+        tracing::trace!(target = "cfd", time_left_in_cfd = %hours_left, "Calculating hours to extend in rollover");
```

Comment:
Where are we printing `hours_to_extend` and can we combine these two log messages?

Reply:
That was my original approach, but we would have to calculate the value of `hours_left` again just for logging. I felt like this information is lower level than the other values we print, so outputting it separately on the trace level made sense to me.

Reply:
Here's the other log:

https://github.com/itchysats/itchysats/blob/3d749ea2767a4279b4ed96b0fa1a262f5743c7a7/model/src/cfd.rs#L921-L927